torch/onnx/symbolic_opset9.py

```diff
@@ -2541,8 +2541,22 @@ def _any(g, input):
     input_sum = sym_help._reducesum_helper(g, input, keepdims_i=0)
     return gt(g, input_sum, g.op("Constant", value_t=torch.LongTensor([0])))
 
-def _all(g, input):
-    return g.op("Not", _any(g, g.op("Not", input)))
+def _all(g, *args):
+    # aten::all(Tensor self)
+    if len(args) == 1:
+        return g.op("Not", _any(g, g.op("Not", args[0])))
```

Comment:
nit: update _any and _all to have one reuse the other and both benefit from this update.

Reply:
Good catch, added that